build/yaml/samples/named-port-sample/rule-out-rule-applied-to-with-dst-with-dup-port-multi.yml

```diff
@@ -0,0 +1,108 @@
+# DB
+# first run
+#  kubectl label ns db2 role=db
+#  test: kubectl exec -ti web-ip -n web -- nc db-ip 3306 -v
+#  kubectl exec -ti web-ip -n web -- nc db-ip 1234 -v
+#  nc db-ip 1234 -v
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mysql-6
+  namespace: db
+  labels:
+    role: db
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      role: db
+  template:
+    metadata:
+      labels:
+        role: db
+    spec:
+      containers:
+        - image: "harbor-repo.vmware.com/dockerhub-proxy-cache/gophernet/netcat:latest"
+          command: ["nc", "-l", "-p", "3306"]
+          name: mysql-6
+          ports:
+            - containerPort: 3306
+              name: mysql-port-6
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mysql-6
+  namespace: db2
+  labels:
+    role: db
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      role: db
+  template:
+    metadata:
+      labels:
+        role: db
+    spec:
+      containers:
+        - image: "harbor-repo.vmware.com/dockerhub-proxy-cache/gophernet/netcat:latest"
+          command: ["nc", "-l", "-p", "1234"]
+          name: mysql-6
+          ports:
+            - containerPort: 1234
+              name: mysql-port-6
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: tcp-deployment-6
+  namespace: web
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      role: web
+  template:
+    metadata:
+      labels:
+        role: web
+    spec:
+      hostname: web-deployment
+      containers:
+        - image: "harbor-repo.vmware.com/dockerhub-proxy-cache/gophernet/netcat:latest"
+          command: ["nc", "-l", "-p", "4321"]
+          name: web-6
+          imagePullPolicy: IfNotPresent
+---
+apiVersion: nsx.vmware.com/v1alpha1
+kind: SecurityPolicy
+metadata:
+  name: named-port-policy-6
+  namespace: web
+spec:
+  priority: 10
+  appliedTo:
+    - podSelector:
+        matchLabels:
+          role: web
+  rules:
+    - direction: out
+      action: allow
+      appliedTo:
+        - podSelector:
+            matchLabels:
+              role: web
+      destinations:
+        - podSelector:
+            matchLabels:
+              role: db
```

Comment:
I guess we can remove L100-L102 because its meaning is "select the pods with label role=db in the same namespace", which can not select the pods in namespace db or db2. There's similar issue in rule-out-rule-applied-to-with-dst-with-dup-port.yml